cmd/guacgql/cmd/ingest.go

```diff
@@ -155,8 +156,8 @@ func ingestSLSA(ctx context.Context, client graphql.Client) {
 		BuildType:     "Test:SLSA2",
 		SlsaPredicate: predicate,
 		SlsaVersion:   "v1",
-		StartedOn:     time.Now(),
-		FinishedOn:    time.Now().Add(10 * time.Second),
+		StartedOn:     &startTime,
+		FinishedOn:    &finishTime,
```

Comment:
Let's remove them from one of the fields, this way we'd be notified when we accidentally make these fields mandatory again

Reply:
done. Removed it from one test and left both the start and end time for the other.